src/_includes/assets/js/banner.js

```diff
@@ -28,12 +28,12 @@ document.addEventListener('DOMContentLoaded', () => {
   const bannerDonorLink = `{% t 'links:banner.authenticated-donor' %}`;
 
   // eslint-disable-next-line no-undef
-  if (isAuthenticated) {
+  if (isAuthenticated && !isDonor) {
     bannerTextNode.innerHTML = bannerAuthText;
     bannerAnchor.href = bannerAuthLink;
     bannerAnchor.setAttribute('text-variation', 'authenticated');
     // eslint-disable-next-line no-undef
-  } else if (isDonor) {
+  } else if (isAuthenticated && isDonor) {
```

Comment:
Pretty sure they can't be a donor without being authenticated.

Reply:
While it simplifies that way, I like things explicitly spelled out.

Reply:
In that case, we should nest it:

```js
if (isAuthenticated) {
  bannerTextNode.innerHTML = isDonor ? bannerDonorText : bannerAuthText
}
else {
  bannerTextNode.innerHTML = bannerDefaultText
}
```

(Or reorder the whole thing so the default is the initial value instead of the fallback)

Reply:
I agree it reads better.